### ISSUE TITLE
Default responseTimeout=0 Causes Zowe CLI Error

### ISSUE THREAD

**Issue by dkelosky** [zowe/vscode-extension-for-zowe#1016]
**Is your feature request related to a problem? Please describe.**

If I create a profile in Zowe Explorer and bypass this prompt by hitting enter:
![image](https://user-images.githubusercontent.com/14940561/93247397-3ebf5580-f75c-11ea-854d-4eb5467bcd74.png)

My profile is created  with:
`responseTimeout: 0`

Then if I run a command like `zowe files download ds "some.data.set(member)" --zosmf-p ze-profile` where `ze-profile` is a profile I created with Zowe Explorer, I get the following error on the CLI:
```
Invalid numeric value specified for option:
--responseTimeout

You specified:
0

Value must be between 5 and 600 (inclusive)
```

**Describe the solution you'd like**

Perhaps Zowe Explorer should create the profile with a default value.  

**Describe alternatives you've considered**

**Additional context**

**Comment by zFernand0** [comment]
> Perhaps Zowe Explorer should create the profile with a default value.

or better yet, ZE could avoid adding optional fields to the profile if the user didn't specify a value for it. 😋

**Comment by dastagg** [comment]
Just so you know:

I can confirm that I have seen (a few) Profiles created through the Zowe Explorer Interface that did have:
responseTimeout: 0
as a parameter in their Profile.

But today( Oct 1, 2020) , using Zowe Explorer 1.9.0:
I created a profile.
It did ask for:
The base path for your API mediation layer instance
The encoding for download and upload of files

But it did not ask for:
The maximum amount of time.

By not entering any information and hitting "Enter" for each of the two last options it created this profile:
```
cat Test3.yaml 
host: 192.168.2.5
port: 10443
user: Test3
password: test3
rejectUnauthorized: false
encoding: 0
```
By not entering any information and hitting "Esc" for each of the two last options it created this profile:
```
$ cat Test4.yaml 
host: 192.168.2.6
port: 10043
user: Test4
password: test4
rejectUnauthorized: false
```

(yeah, I don't have zowe secure installed on this computer.)

**Comment by jellypuno** [comment]
Closing issue. Fix is merged in v1.12.x
